.pre-commit-config.yaml

```diff
@@ -0,0 +1,22 @@
+repos:
+-   repo: https://gitlab.com/pycqa/flake8
+    rev: 3.8.1
+    hooks:
+    - id: flake8
+      pass_filenames: true
+      # this seems to need to be here in addition to setup.cfg
+      exclude: cibuildwheel/resources/
```

Comment:
Why is this? This is set in `setup.cfg`, so why doesn't it work; does it have to do with this `pass_filenames`? Can't we run this the same way as on the CI?

Reply:
there is problem in pre commit and it does not handle configuration from setup.cfg

Reply:
Yes, I read that comment. But is it a known bug? Or does it get solved it you don't set `pass_filenames`? I don't want to repeat all configuration multiple times, if possible.

Reply:
It is not solved

Reply:
Ah, too bad :-( Do you have a reference/link to add, so that it can later be checked whether it is fixed?

And are the other lines of `setup.cfg` honored, like `ignore = E501,W503`?

Reply:
https://github.com/pre-commit/pre-commit-hooks/issues/112

pre commit check files and pass it to proper task. So it ignore all settings connected with include/exclude path and use others.

Reply:
Thanks! That makes sense. I guess that's OK, then, if it's just about file selection and if other changes are propagated.